src/test/java/uk/gov/hmcts/reform/sscs/service/HearingsServiceTest.java

```diff
@@ -204,7 +209,12 @@ void processHearingWrapperUpdate() {
                 .willReturn(HmcUpdateResponse.builder().build());
 
         wrapper.setState(UPDATE_HEARING);
-        wrapper.getCaseData().getSchedulingAndListingFields().setActiveHearingId(HEARING_REQUEST_ID);
+        wrapper.getCaseData()
+            .setHearings(new ArrayList<>(Collections.singletonList(Hearing.builder()
```

Comment:
minor/optional: we don't need to wrap this in an arrayList

Reply:
![image](https://user-images.githubusercontent.com/18538046/173392875-0270e84d-55cd-4fa8-9983-bbd6f509d1c6.png)
You can't add a hearing if you do that